src/components/HighlightCard/HighlightCard.test.jsx

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect } from "vitest";
+import { render, screen } from "@testing-library/react";
+import HighlightCard from "./HighlightCard";
+
+const renderHighlightCard = (props = {}) => {
+  const defaultProps = {
+    title: "Visibility",
+    data: "10000",
+    unit: "miles",
+    other: false,
+    range: 80,
+  };
+  return render(<HighlightCard {...defaultProps} {...props} />);
+};
+
+describe("<HighlightCard />", () => {
+  it("renders default props", () => {
+    renderHighlightCard();
+    expect(screen.getByText("Visibility")).toBeInTheDocument();
+    expect(screen.getByText("10000")).toBeInTheDocument();
+    expect(screen.getByText("miles")).toBeInTheDocument();
+  });
+
+  it("doesn't renders the element when other is false", () => {
+    renderHighlightCard({ other: false });
+    expect(screen.queryByText("WSW")).toBeNull();
```

Comment:
Usually I use `.not.toBeInTheDocument()`, which is more clear that's `not` there, but this is fine as well.